.changeset/pink-islands-return.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lion/ui': patch
+---
+
+[BREAKING-CHANGE] [input-stepper] a11y enhancement & added translations
```

Comment:
How exactly is it breaking?

Reply:
Changed a function name, but that was private, so can be changed to none breaking.

Will then directly add it to master :)